acceptance/bundle/templates/default-sql/test.toml

```diff
@@ -0,0 +1,12 @@
+[[Repls]]
+Old = '- (user_name|service_principal_name): \$USERNAME'
+New = '- user_name_OR_service_principal_name: $$USERNAME'
```

Comment:
Can you include a comment explaining why these replacements need to happen? Good for posterity.

Reply:
Good idea. For now, I've disabled bundle/templates in cloud env as there are many differences that deserve their own PR.